src/server/server.js

```diff
@@ -15,6 +16,9 @@ app.use(bodyParser.urlencoded({extended: true}));
 database.sync({alter:true});
 
 unauthorized(app);
+
+app.use((request,response,next)=>{authVerification.jwtVerify(request,response,next)});
```

Comment:
send app to middleware then use there jwt now need to do here. refer the repo which in provide you.